generator-elide/generators/app/prompt.js

```diff
@@ -0,0 +1,76 @@
+const generator = require('./generator');
+const model = require('./model');
+const misc = require('./misc.js');
+
+// Main prompt when running 'yo elide'
+const prompting = (yo) => {
+  yo.prompt([{
+    type: 'list',
+    name: 'command',
+    message: 'Choose an option below',
+    choices: [ 'Try an example', 'Create a new project', 'Info & Contact' ]
+  }]).then((answers) => {
+    switch (answers.command) {
+      case 'Try an example':
```

Comment:
Can we use string constants here? That will lessen the likelihood of fat-fingering this in the future. Namely, something like:

```
const TRY = 'Try an example';
const CREATE = 'Create a new project';
const INFO = 'Info & Contact';
```

Then you can use those vars both here in the `switch` statement and above in the `choices` list.